Dockerfile

```diff
@@ -1,8 +1,8 @@
 FROM jenkins/jnlp-slave:3.16-1-alpine
 MAINTAINER Infrastructure team <team-infrastructure@newstore.com>
 
-ENV PG_MAJOR 9.6
-ENV PG_VERSION 9.6.5
+ENV PG_MAJOR 10
+ENV PG_VERSION 10.0
```

Comment:
must be 10.6

Reply:
already tested with 10.6 and it work 
_/home/jenkins # pg_dump -V
pg_dump (PostgreSQL) 10.6
/home/jenkins #_

Reply:
Fied in 4f2d03875a55896ce8eab3e322cde949b756168b